config/locales/en/views.yml

```diff
@@ -4,6 +4,17 @@ en:
       intro: GOV.UK uses cookies to make the site simpler.
       find_out_more: Find out more about cookies.
   booking_requests:
+    booking_calendar:
+      today: Today
+    booking_calendar_legend:
+      visit_days: Visit Days
+      non_visit_days: Non-visit Days
+    slot_picker_time_slots:
+      visit_in_the_past: It is not possible to book a visit in the past.
+      visit_today: It is not possible to book a visit on this day.
+      bookability_window: >-
+        You can only book a visit in the next {{ daysInRange }} days.
```

Comment:
Putting Mustace in here feels a bit odd. I think it would be more consistent to do something like:

```yaml
bookability_window: >-
  You can only book a visit in the next %{days} days.
```

and call it with the interpolated value:

```ruby
t('.bookability_window', days: '{{ daysInRange }}')
```

That would at least sequester all the Mustache in the template files.